openapi-v1.yaml

```diff
@@ -2041,6 +2041,15 @@ definitions:
           type: string
           x-omitempty: true
 
+  TileDBCloudUDFRuntimeProfile:
```

Comment:
Will this ever apply to server-side SQL which also runs in a Docker image?

Reply:
Yes

Reply:
@Shelnutt2 great! In that case @thetorpedodog maybe we could rename this. Or, accept that `UDF` sometimes means "for UDF and SQL" ... :)

Reply:
`ResourceClass`? `ContainerSize`? I'm fine with the imprecise name but those are a couple ideas we can take if we like them.

Reply:
oo!! I like `ResourceClass`!! Good room for future expansion there :). (`ContainerSize` is fine for now but might feel too narrow if we latter add some dimension other than size in the future ...)

Reply:
or `ResourceProfile`, since we use "server profile" in the UI in notebooks.

Reply:
@thetorpedodog even better! :)

Reply:
We reserve `ResourceProfile` but it will be used if we decide that `runtime_profile` should be an enum. If we go with strings, it will be used in documentation